sdk/include/cardboard.h

```diff
@@ -42,6 +42,14 @@ typedef enum CardboardEye {
   kRight = 1,
 } CardboardEye;
 
+/// Enum to distinguish the possible orientation of the viewport
+typedef enum CardboardViewportOrientation {
+  kCardboardViewportOrientationLandscapeLeft = 0,
```

Comment:
Please add a comment to each possible value. This will improve the reference docs.